src/Diagnostics.RuntimeHost/Properties/launchSettings.json

```diff
@@ -3,7 +3,8 @@
     "Diagnostics.RuntimeHost": {
       "commandName": "Project",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "AzureServicesAuthConnectionString": "RunAs=Developer; DeveloperTool=VisualStudio"
```

Comment:
What does this change do?

Reply:
It's an experimental change for speeding up the local start time. It slightly improves.